docs/architecture/deep-dives/passkeys/implementations/relying-party/prf.md

```diff
@@ -0,0 +1,101 @@
+# Passkeys for Decryption
+
+When a user logs into their Bitwarden vault, two related but distinct operations take place -
+authentication and decryption.
+
+When using a master password, the password itself serves both of these purposes - authenticating the
+user by verifying the password was entered correctly, and allowing decryption of your data through
+the derivation of an encryption key from the password. For more detail on this, see the
+[Bitwarden Security Whitepaper](https://bitwarden.com/help/bitwarden-security-white-paper/).
+
+Passkeys inherently provide the mechanism for authentication. The `prf` extension introduced in the
+WebAuthn [specification](https://w3c.github.io/webauthn/#prf-extension) provides the framework
+necessary for passkeys to be used for decryption as well.
+
+## What is PRF?
+
+At the most abstract level, account encryption and decryption requires that the user's symmetric key
+be encrypted ("protected") with another key that is available only to the client. For master
+password-based decryption, this is done with the master key, derived through a key derivation
+function from the master password.
+
+In order to use a passkey for account decryption on login, we need such a key available on the
+passkey, with which we can encrypt and decrypt the user's symmetric key. Fortunately, the `prf`
+extension to the WebAuthn spec provides exactly this functionality.
```

Comment:
suggestion: This paragraph seems to be tightly connected to the previous one, consider merging.

suggestion: Simplify. Remove "we" and use "client" which is used earlier in the paragraph
```suggestion
In order to use a passkey for account decryption, the client needs to able to derive such a key from the passkey. Fortunately, the `prf` extension provides exactly this functionality.
```

Reply:
I addressed this with [4ab27ff (#307)](https://github.com/bitwarden/contributing-docs/pull/307/commits/4ab27ff50f7a78620ea68d6f99691c481f24c263).